### DISCUSSION TITLE
接入bk-ci想要徽章，并且可以跳转到最新执行的流水线状况

### DISCUSSION THREAD

**Discussion by liuguai** [Tencent/bk-ci#4762] [Ideas]
想要在readme放置徽章，发现原蓝盾徽章对应的仓库是git_xxxxx ，填入后发现徽章显示正常了，但是跳转过去会发现没有项目的权限，这样工蜂ci的一个优点（只需要管理工蜂权限）就没有了；可以提供我们流水线最新创建的跳转连接吗

**Comment by zanyzhao** [comment]
抱歉，回复晚了。

徽章功能已经在设计，预计月底就上线了
